app/serializers/activitypub/note_serializer.rb

```diff
@@ -33,6 +35,13 @@ def content_map
     { object.language => Formatter.instance.format(object) }
   end
 
+  def replies
+    ActivityPub::CollectionPresenter.new(
+      type: :unordered,
+      items: object.self_replies(5).map(&:uri)
```

Comment:
I guess it's odd to hardcode limit this to 5. For example, there is this self-replies thread of every "good morning" since January 1st: https://mastodon.technology/@ashfurrow/101341748880807263 And it's a lot of items... But only listing 5 of them seems inaccurate, without a way to indicate that there's more.

Reply:
Maybe it would be better to add something making explicit that the collection is incomplete, indeed.
But for the purposes of this PR, it doesn't need to list all replies, especially not replies that are not immediate replies.

Reply:
I can't find a better attribute than `replies` for this, and I can't find an existing `Collection` attribute to indicate it's incomplete. But I think it's fine this way, if not completely satisfying.

Reply:
Hm, while I do understand your suggestion of using a paginated `Collection` not restricted to a hardcoded limit of 5 self-replies, I think this would make the code much more complex for no real benefit.
Indeed, going with your logic, `replies` should not be limited to self-replies either, so we would need to have up to 5 self-replies in the first page, *then* have the following page include both self-replies and other replies, excluding the 5 self-replies, etc. That would also require a new URL scheme, etc.
The `replies` collection is never going to be truly exhaustive anyway, we might exclude muted users, private posts, etc.